### ISSUE TITLE
Publish `wpe-headless` plugin via GitHub Actions

### ISSUE THREAD

**Issue by blakewilson** [wpengine/faustjs#479]
We should publish the `wpe-headless` plugin via GitHub Actions and move away from CircleCI as the rest of our CI/CD process is now using GitHub Actions

**Comment by justlevine** [comment]
Can it also push the plugin to Packagist.org ? Manually downloading and installing new versions of wpe-headless seems kind of ...archaic.

**Comment by blakewilson** [comment]
@justlevine Yes absolutely. We will be releasing the plugin on the official WP repo, as well as packagist.org once we get to an official v1.

**Comment by justlevine** [comment]
I understanding on waiting until 1.0 to release in the WP repo, but I really hope packagist.org is added sooner, as it's because of the lack of the Wordpress Update flow that a way to update the plugin without manually downloading and replacing in /plugins is needed. Unless you're planning to implement the update mechanism used in Atlas Content Modeler.... is this better in a separate ticket?

**Comment by blakewilson** [comment]
@justlevine, maybe I misread your comment, but the `wpe-headless` plugin does have a built in WordPress update flow, like ACM, where the plugin will prompt you to update if there is a new version present:

![Screen Shot 2021-09-28 at 12 22 10 PM](https://user-images.githubusercontent.com/5946219/135135388-b9b3df66-db49-40c5-adff-efbe66ae8ee3.png)
